stomp_core/src/stomp.cpp

```diff
@@ -590,26 +592,25 @@ bool Stomp::computeNoisyRolloutsCosts()
 
 bool Stomp::computeRolloutsStateCosts()
 {
-
-  bool all_valid = true;
   bool proceed = true;
+
+  #pragma omp parallel for shared(proceed)
   for(auto r = 0u ; r < config_.num_rollouts; r++)
   {
-    if(!proceed_)
+    if(proceed)
```

Comment:
The ```proceed``` variable is never changed and so there is no point in checking it.  The original code changed the ```proceed_```  atomic boolean member from outside the function so that the optimization could be stopped upon user request.  This change won't allow to cancel out of an ongoing plan.